src/components/new-members/new-members.module.scss

```diff
@@ -6,6 +6,21 @@
   justify-content: center;
 }
 
+.newUser {
+  pointer-events: none;
+  display: flex;
+  padding: 20px 10px;
+  background: white;
+  max-width: 270px;
+  border-radius: 5%;
+  transition: 0.38s box-shadow ease-out;
+  cursor: none;
+  flex: 0 150px;
+  height: 160px;
+  overflow: hidden;
+  margin: auto;
```

Comment:
Is this really required here ? Which other `property` can we use here ?

Reply:
yeah, the margin is needed. We need the same CSS that we have used earlier for NewUser.

Reply:
Okay. But we ought to use `padding` instead of `margin` because more often, adding `margin` leads to breaking off `containers`.

Reply:
Ok. I changed to `padding` now instead of `margin`. Thanks.